src/plugins/console/public/application/components/settings_modal.tsx

```diff
@@ -23,13 +23,19 @@ import {
   EuiModalHeader,
   EuiModalHeaderTitle,
   EuiSwitch,
+  EuiSelect,
+  EuiFlexGroup,
+  EuiFlexItem,
 } from '@elastic/eui';
 
 import { DevToolsSettings } from '../../services';
-import { EuiRefreshInterval } from './refresh_interval';
 
 export type AutocompleteOptions = 'fields' | 'indices' | 'templates';
 
+const MILLISECONDS_IN_MINUTE = 60000;
```

Comment:
nit: what do you think about replacing these constants and the `intervalOptions` definition with something more compact that uses pluralization:

```typescript
const PRESETS_IN_MINUTES = [1, 5, 10];
const intervalOptions = PRESETS_IN_MINUTES.map(value => ({
    value: value * 60000,
    text: i18n.translate('console.settingsPage.refreshInterval.timeInterval', {
        defaultMessage: '{value} {value, plural, one {minute} other {minutes}}',
        values: { value },
    }),
}));
```